.github/workflows/notify.js

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const needsTestFiles =
+    'We have detected that your changes do not add or modify test files. Please add or update tests or indicate why tests are not needed.'
```

Comment:
```suggestion
    'This pull request modifies files in src/ but no tests were added/updated. Confirm whether tests should be added or ensure the PR description explains why tests are not required.'
```

Ideally this comment would be skipped if the PR description already mentions "test"?